packages/legend-studio/src/stores/editor-state/element-editor-state/mapping/MappingEditorState.ts

```diff
@@ -766,23 +766,30 @@ export class MappingEditorState extends ElementEditorState {
       );
       return;
     }
+    if (mappingElement.parent !== this.element) {
+      this.editorStore.openElement(mappingElement.parent);
+    }
+    const currentOpenedMappingEditorState = this.editorStore
```

Comment:
I think we can use `getCurrentEditorState()` here. Also `currentOpenedMappingEditorState` --> `currentMappingEditorState`